samples/cpp/ngraph_function_creation_sample/main.cpp

```diff
@@ -90,7 +62,7 @@ ov::runtime::Tensor ReadWeights(const std::string& filepath) {
     std::ifstream weightFile(filepath, std::ifstream::ate | std::ifstream::binary);
```

Comment:
Function name is ReadWeights. Can we align naming style with OpenVINO runtime?

Reply:
we'll do in separate PRs